samples/src/main/java/com/example/bigquery/AddEmptyColumn.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquery;
+
+// [START bigquery_add_empty_column]
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.Field;
+import com.google.cloud.bigquery.FieldList;
+import com.google.cloud.bigquery.LegacySQLTypeName;
+import com.google.cloud.bigquery.Schema;
+import com.google.cloud.bigquery.StandardTableDefinition;
+import com.google.cloud.bigquery.Table;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class AddEmptyColumn {
+
+  public static void runAddEmptyColumn() {
+    // TODO(developer): Replace these variables before running the sample.
+    String datasetName = "MY_DATASET_NAME";
+    String tableId = "MY_TABLE_NAME";
+    addEmptyColumn(datasetName, tableId);
+  }
+
+  public static void addEmptyColumn(String datasetName, String tableId) {
+    try {
+      // Initialize client that will be used to send requests. This client only needs to be created
+      // once, and can be reused for multiple requests.
+      BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
+
+      Table table = bigquery.getTable(datasetName, tableId);
+      Schema schema = table.getDefinition().getSchema();
+      FieldList fields = schema.getFields();
+
+      // Create the new field/column
+      String randomColumnName = "new_" + UUID.randomUUID().toString().replace('-', '_');
+      Field newField = Field.of(randomColumnName, LegacySQLTypeName.STRING);
+
+      // Create a new schema adding the current fields, plus the new one
+      List<Field> fieldList = new ArrayList<Field>();
+      for (Field f : fields) {
+        fieldList.add(f);
+      }
+      fieldList.add(newField);
+      Schema newSchema = Schema.of(fieldList);
+
+      // Update the table with the new schema
+      table.toBuilder().setDefinition(StandardTableDefinition.of(newSchema)).build().update();
```

Comment:
nit: Consider splitting this into two actions to make more clear to users what's happening:
```java
// Update the table definition to include the new schema 
updatedTable = table.toBuilder().setDefinition(StandardTableDefinition.of(newSchema)).build();
// Push updates to the BigQuery service
updatedTable.update(); 
```